src/pages/index.tsx

```diff
@@ -1,43 +1,111 @@
-            <Date />
-            <EventsContainer />
-            <Footer />
-            <style jsx>{`
-                .container {
-                    min-height: 100vh;
-                    display: flex;
-                    flex-direction: column;
-                    justify-content: center;
-                    align-items: center;
-                }
-            `}</style>
+import { Event } from "utils/types";
+import { useRouter } from "next/router";
+import withWidth from "@material-ui/core/withWidth";
+import { getEvents } from "server/actions/Event";
+import constants from "utils/constants";
+import { Button, Container, createStyles, Grid, makeStyles, Theme } from "@material-ui/core";
+import CoreTypography from "src/components/core/typography";
+
+interface Props {
+    events: Event[];
+    width: string;
+}
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        jumbotron: {
+            width: "100%",
+            height: "65vh",
+            backgroundColor: theme.palette.primary.main,
+            position: "relative",
+            top: 0,
+        },
+    })
+);
 
-            <style jsx global>{`
-                html,
-                body {
-                    padding: 0;
-                    margin: 0;
-                    font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen, Ubuntu, Cantarell,
-                        Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
-                }
-                * {
-                    box-sizing: border-box;
-                }
-            `}</style>
+const Home: NextPage<Props> = ({ events, width }) => {
+    const classes = useStyles();
+
+    return (
+        <div style={{ width: "100%" }}>
+            <div className={classes.jumbotron}>
+                <Grid
+                    container
+                    spacing={3}
+                    direction="row"
+                    justify="center"
+                    style={{ width: "100%", paddingTop: width == "xs" ? "10%" : "" }}
+                >
+                    <Grid
+                        item
+                        xs={12}
+                        sm={7}
+                        lg={6}
+                        style={{
+                            display: "flex",
+                            flexDirection: "column",
+                            justifyContent: "center",
+                            paddingLeft: "10%",
+                        }}
+                    >
+                        <CoreTypography variant="h1" style={{ color: "white" }}>
+                            Upcoming events
+                        </CoreTypography>
+                        <CoreTypography variant="h3" style={{ fontWeight: "normal", color: "white", marginBottom: 30 }}>
+                            Join us for a workday!
+                        </CoreTypography>
+                        <DDate />
```

Comment:
You can remove the Dummy Date here. I just used that to get the exact format that MUI used for formatting the input date.